iree/turbine/kernel/compiler/host_codegen.py

```diff
@@ -80,13 +80,22 @@ def isolated_test_call(
             dispatch = SymbolRefAttr.get([exe.sym_name.value, entrypoint])
             entrypoints = ArrayAttr.get([dispatch])
 
+            buffer_binding_count = len(sig.kernel_buffer_bindings)
+            offset_to_out_binding = len(sig.kernel_buffer_input_bindings)
```

Comment:
minor: how about renaming this to input_binding_count ?

Reply:
done!